MeshLib/MeshEditing/ProjectPointOnMesh.cpp

```diff
@@ -0,0 +1,75 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "GeoLib/AnalyticalGeometry.h"
+#include "MathLib/Point3d.h"
+
+#include "MeshLib/Elements/Quad.h"
+#include "MeshLib/Elements/Tri.h"
+#include "MeshLib/MeshSearch/MeshElementGrid.h"
+
+namespace MeshLib
+{
+namespace ProjectPointOnMesh
+{
+
+/// Returns the element in which the given node is located when
+/// projected onto a mesh, or nullptr if no such element was found.
```

Comment:
Doxygen hasn't finished yet, but the function documentation appears twice in the code, once here, once in the header file; it could be that it will show up twice in the doxygen's function documentation. Needs a quick check, when doxygen is done.

Reply:
Removed the duplicate comments.